bin/genparser

```diff
@@ -0,0 +1,162 @@
+#!/bin/sh
+#
+# Copyright by The HDF Group.
+# Copyright by the Board of Trustees of the University of Illinois.
```

Comment:
Just checking:  Do you want to include this Copyright line?

Reply:
I'm just copying the HDF4 lines. Do we need to apply the HDF5 rules to HDF4?

Reply:
> Just checking: Do you want to include this Copyright line?

We need to keep the UofI copyright line if the file was created before 2006.